pyoptsparse/pyALPSO/pyALPSO.py

```diff
@@ -25,9 +26,7 @@ class ALPSO(Optimizer):
     - pll_type -> STR: ALPSO Parallel Implementation (None, SPM- Static, DPM- Dynamic, POA-Parallel Analysis), *Default* = None
     """
 
-    def __init__(self, raiseError=True, options={}):
-        from . import alpso
-
+    def __init__(self, options={}):
```

Comment:
I will not use an empty dictionary as default value for options here (see [here](https://stackoverflow.com/questions/26320899/why-is-the-empty-dictionary-a-dangerous-default-value-in-python))

Reply:
This comment has nothing to do with this PR.

Reply:
Yes true but you could also solve this problem :-) 

Or open another PR where you solve that. Because it can give bugs :-)